bufferParameters:
  maxSize : 1e6
  batchSize: 256

policy:
  env : FetchReach-v1
  seed: 0
  hiddenSizes: [256, 256]
  steps_per_epoch : 5000
  epochs : 5
  gamma : 0.99
  polyak : 0.95
  lr : 1e-3
  alpha : 0.2
  start_steps : 10000
  max_ep_len : 50
